### ISSUE TITLE
Where is the source of truth?

### ISSUE THREAD

**Issue by martinthomson** [mnot/ietf-repo-data#3]
`github-issue-label` being in the document has an odd smell.  As a highly specific thing with a triple-barrel name, it suggests that maybe there is other document-specific meta-information that you might want to carry.  And I can see how you might want to build a tool that presents issues on a specific spec rather than the whole repo.

But my first thought was that you might want to use that information in generating the notice you insert in the document.  For which you need far more information than is presented here: including information from the "primary" repo so that you can include information about the working group, such as the location of the mailing list.

So where would such a tool get that information?  I suspect that the right place for that information is `ietf.json`, and the conclusion might be that building a draft would require input from `ietf.json` files from this and other repositories.

That all suggests to me that embedding information with extremely narrow use cases in the document itself is not the right answer and that the information should be in the metadata file.

Or, if it is in the draft, we need a way to think about what sort of information is needed for drafts and to structure that information more rationally than just having ad-hoc insertions of fields.

**Comment by mnot** [comment]
Yes, but that would require putting spec-specific information in `ietf.json`. I want to require people to touch that file as little as possible.

What kind of structure are you thinking of, beyond k/v pairs?
